documentation/modules/managing/proc-pausing-reconciliation.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// assembly-management-tasks.adoc
+
+[id='proc-pausing-reconciliation-{context}']
+
+= Pausing reconciliation of custom resources
+
+Sometimes it is useful to pause the reconciliation of custom resources managed by Strimzi Operators,
+so that you can perform fixes or make updates.
+If reconciliations are paused, any changes to custom resources are ignored by Strimzi Operators until the pause ends.
+
+If you want to pause reconciliation of a custom resource, set the `pause-reconciliation` annotation to `true` in its configuration.
+This instructs the appropriate Operator to pause reconciliation of the custom resource.
+For example, you can apply the annotation to the `KafkaConnect` resource so that reconciliation by the Cluster Operator is paused.
+
+You can also _create_ a custom resource with the pause annotation enabled.
+The custom resource is created, but it is ignored.
+
+To resume reconciliation, you can set the annotation to `false`, or remove the annotation.
```

Comment:
Should we add this as a step into the procedure? It seems a bit lost here.

Reply:
Yeah. I moved it out of the procedure, but I think maybe as a next step works best

![image](https://user-images.githubusercontent.com/47596553/110928664-0d949c00-831f-11eb-9faa-c27efa2b5d8a.png)